qutip/solver/solver_base.py

```diff
@@ -46,6 +46,8 @@ def __init__(self, rhs, *, options=None):
             self.rhs = QobjEvo(rhs)
         else:
             TypeError("The rhs must be a QobjEvo")
+        self.sys_dims = rhs.dims[0]
+        self.evolve_dm = rhs.issuper
```

Comment:
I'm guessing `evolve_dm` means that a density matrix can be passed to `.run()` and similar, but the `evolve_dm` attribute doesn't appear to be used by anything in this PR yet.